.github/ISSUE_TEMPLATE/feature.yaml

```diff
@@ -0,0 +1,11 @@
+name: Feature
```

Comment:
Should we mention something about OFEPs for features that go beyond the scope of FlagD?

Reply:
I think that's a good idea. I'll open a PR soon.